apps/antalmanac/src/stores/calendarizeHelpers.ts

```diff
@@ -10,7 +10,7 @@ const FINALS_WEEK_DAYS = ['Sat', 'Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri'];
 
 export function getLocation(location: string): Location {
     const [building = '', room = ''] = location.split(' ');
-    return { building, room };
+    return { building, room, days: null };
```

Comment:
As mentioned above, days should just be optional and it can be omitted from the object altogether (which is synonymous with being undefined).

```suggestion
    return { building, room };
```